src/test/resources/features/dashboardsTable.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Dashboards feature
+  As an user
+  I want to see a dashboards table on Dashboard Page with correct columns and items on it
+
+  Background: : User is able to login to Report Portal
+    Given User logins to Report Portal
+
+  @ui
+  Scenario: User is able to see items on dashboards table
+    Given Dashboards page is opened
+    And Click project widget
+    When Choose project name
```

Comment:
Like in the case of login and login service you can make a Service layer and combine there two steps for moving to dashboard page. But in the case of BDD, the step definitions layer can be used as a Service layer

Reply:
Thanks. Done.